Resources/Computing/GlobusComputingElement.py

```diff
@@ -0,0 +1,283 @@
+########################################################################
+# File :   GlobusComputingElement.py
+# Author : A.S.
+########################################################################
+
+""" Globus Computing Element
+
+   Allows direct submission to Globus Computing Elements with a SiteDirector Agent
+
+"""
+
+__RCSID__ = "$Id$"
+
+from DIRAC.Resources.Computing.ComputingElement          import ComputingElement
+from DIRAC.Core.Utilities.Grid                           import executeGridCommand
+from DIRAC                                               import S_OK, S_ERROR
+
+from DIRAC.ConfigurationSystem.Client.Helpers.Registry   import getGroupOption
+from DIRAC.FrameworkSystem.Client.ProxyManagerClient     import gProxyManager
+from DIRAC.WorkloadManagementSystem.DB.PilotAgentsDB     import PilotAgentsDB
+
+
+import os, tempfile
+
+CE_NAME = 'Globus'
+MANDATORY_PARAMETERS = [ 'Queue' ]
+
+class GlobusComputingElement( ComputingElement ):
+  """Globus computing element class
+  implementing the functions jobSubmit, getJobOutput """
+
+  #############################################################################
+  def __init__( self, ceUniqueID ):
+    """ Standard constructor.
+    """
+    ComputingElement.__init__( self, ceUniqueID )
```

Comment:
Because old and new style classes are different, and old-style classes don't have super
```
class ClassN(object):
    def __init__(self):
        self.var = "N1"

class ClassNI(ClassN):
    def __init__(self):
        super(ClassNI, self).__init__()

class ClassO:
    def __init__(self):
        self.var = "N1"

class ClassOI(ClassO):
    def __init__(self):
        super(ClassOI, self).__init__()

if __name__ == "__main__":
    n = ClassNI()
    o = ClassOI()
```
Leads to
```
Traceback (most recent call last):
  File "OldvsNew.py", line 19, in <module>
    o = ClassOI()
  File "OldvsNew.py", line 15, in __init__
    super(ClassOI, self).__init__()
TypeError: must be type, not classobj
```
See for example: http://stackoverflow.com/questions/9698614/super-raises-typeerror-must-be-type-not-classobj-for-new-style-class